policy/modules/system/init.te

```diff
@@ -303,6 +303,7 @@ ifdef(`init_systemd',`
 
 	allow init_t init_runtime_t:sock_file manage_sock_file_perms;
 
+	allow init_t daemon:process noatsecure;
```

Comment:
This should be a separate patch. Also it wouldn't be here, but rather in `init_domain`, `init_daemon_domain` etc.

Reply:
Moved into init_daemon_domain and into patch #752